### PR TITLE
various bug fixes for the v8.2.2_merge branch

### PR DESCRIPTION
**Pull request by joeolson42**
The following PR contains:

1. Fixes for MMM's MYNN pre & post (Thanks Tanya!)
2. Some format changes to bring our code closer to MMM's code (some of it making it uglier--adding back spaces)
3. removing some lingering (very minor) bugs

### CODE REVIEW

File: src/core_atmosphere/Registry.xml [hunk 1]
```diff
@@ -599,11 +599,11 @@
...
-		<stream name="restart"
-                        type="input;output"
-                        filename_template="restart.$Y-$M-$D_$h.$m.$s.nc"
-                        input_interval="initial_only"
-                        output_interval="1_00:00:00"
+		<stream name="restart" 
+                        type="input;output" 
+                        filename_template="restart.$Y-$M-$D_$h.$m.$s.nc" 
+                        input_interval="initial_only" 
+                        output_interval="1_00:00:00"
```
Comment (tanyasmirnova) [inline src/core_atmosphere/Registry.xml:606]: @joeolson42 Joe, why d owe need white spaces at the end of lines?

File: src/core_atmosphere/physics/mpas_atmphys_driver.F [hunk 2]
```diff
@@ -196,20 +195,20 @@ subroutine physics_driver(domain,itimestep,xtime_s)
...
     call mpas_pool_get_dimension(block%dimensions,'cellSolveThreadEnd',cellSolveThreadEnd)
 
-    call mpas_log_write( 'allocate arrays shared by all physics parameterizations:')
+    !allocate arrays shared by all physics parameterizations:
     call allocate_forall_physics(block%configs)
 
     !physics prep step:
     time_lev = 1
-    call mpas_log_write( 'MPAS to physics' )
+
```
Comment (tanyasmirnova) [inline src/core_atmosphere/physics/mpas_atmphys_driver.F:203]: @joeolson42 I think it is nice to have these print statements while debugging the code. Could we keep these lines with mpas_log_write?

File: src/core_atmosphere/physics/mpas_atmphys_driver.F [hunk 3]
```diff
@@ -196,20 +195,20 @@ subroutine physics_driver(domain,itimestep,xtime_s)
...
 !$OMP PARALLEL DO
     do thread=1,nThreads
        call MPAS_to_physics(block%configs,mesh,state,time_lev,diag,diag_physics, &
                             cellSolveThreadStart(thread),cellSolveThreadEnd(thread))
     end do
 !$OMP END PARALLEL DO
 
-    call mpas_log_write( ' call to cloud scheme: ')
+    !call to cloud scheme:
```
Comment (tanyasmirnova) [inline src/core_atmosphere/physics/mpas_atmphys_driver.F:211]: same as on line 204

File: src/core_atmosphere/physics/mpas_atmphys_driver.F [hunk 4]
```diff
@@ -220,7 +219,7 @@ subroutine physics_driver(domain,itimestep,xtime_s)
 !$OMP END PARALLEL DO
     endif
 
-    call mpas_log_write( ' call to short wave radiation scheme: ')
+    !call to short wave radiation scheme:
```
Comment (tanyasmirnova) [inline src/core_atmosphere/physics/mpas_atmphys_driver.F:222]: same here

File: src/core_atmosphere/physics/mpas_atmphys_driver.F [hunk 5]
```diff
@@ -233,7 +232,7 @@ subroutine physics_driver(domain,itimestep,xtime_s)
 !$OMP END PARALLEL DO
     endif
 
-    call mpas_log_write( ' call to long wave radiation scheme: ')
+    !call to long wave radiation scheme:
```
Comment (tanyasmirnova) [inline src/core_atmosphere/physics/mpas_atmphys_driver.F:235]: same here

File: src/core_atmosphere/Registry.xml (hunk 1 above)
Comment (joeolson42) [reply src/core_atmosphere/Registry.xml:606]: To match NCAR's code

File: src/core_atmosphere/physics/mpas_atmphys_driver.F (hunk 2 above)
Comment (joeolson42) [reply src/core_atmosphere/physics/mpas_atmphys_driver.F:203]: To match NCAR's code. All of your questions have the same answer. Do we want to match NCAR's code or not?

**Review by tanyasmirnova — APPROVED** [review]

**Review by barlage — APPROVED** [review]
- successfully compiled on jet
- successfully ran conus mesoscale_reference + bl_mynn testcase
- successfully ran conus mesoscale_reference testcase

**Pull request merged**